### PR TITLE
fix: standardized repo layer

### PR DESCRIPTION
**Pull request by josephdev9**

### CODE REVIEW

**Review by JosesGabriel — APPROVED** [review]

File: lib/internal/env.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import 'package:flirt/core/domain/repository/secure_storage_repository.dart';
+import 'package:flirt/internal/local_storage.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+Future<String> checkEnvironment(
```
Comment (kabaluyot) [inline lib/internal/env.dart:5]: comment documentation

File: lib/internal/env.dart [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  ISecureStorageRepository storage, {
+  String apiStaging = 'STAGING_QUOTE_API',
+  String apiProduction = 'QUOTE_API',
```
Comment (kabaluyot) [inline lib/internal/env.dart:8]: API_URL=

STAGING_API_URL=

File: lib/core/infrastructures/repository/quote_repository.dart [hunk 3]
```diff
@@ -15,40 +15,40 @@ class QuoteRepository implements IQuoteRepository {
...
-  //   } else {
-  //     _baseURL = dotenv.get('APP_API');
-  //   }
-  // }
+  // Future<String> get _baseURL => checkEnvironment(_storage);
 
   @override
   Future<QuoteResponse> fetchQuote() async {
+    final http.Response response;
     try {
```
Comment (kabaluyot) [inline lib/core/infrastructures/repository/quote_repository.dart:25]: newline

File: lib/internal/env.dart (hunk 2 above)
Comment (kabaluyot) [reply lib/internal/env.dart:8]: API_URL = STAGING

File: lib/core/infrastructures/repository/quote_repository.dart [hunk 4]
```diff
@@ -15,40 +15,40 @@ class QuoteRepository implements IQuoteRepository {
...
-  // @override
-  // Future<void> checkEnvironment() async {
-  //   if (await _storage.read(key: lsEnvironment) != 'production' &&
-  //       dotenv.get('API_ENV') != 'production') {
-  //     _baseURL = dotenv.get('STAGING_APP_API');
-  //   } else {
-  //     _baseURL = dotenv.get('APP_API');
-  //   }
-  // }
+  // Future<String> get _baseURL => checkEnvironment(_storage);
```
Comment (kabaluyot) [inline lib/core/infrastructures/repository/quote_repository.dart:20]: enable

File: lib/internal/repository_utils.dart
Comment (kabaluyot) [inline lib/internal/repository_utils.dart:1]: api.dart

File: lib/internal/utils.dart [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import 'dart:developer';
+
 import 'package:flutter/material.dart';
 
 final Map<String, String> httpRequestHeaders = <String, String>{
```
Comment (kabaluyot) [inline lib/internal/utils.dart:5]: move to api.dart

**Pull request merged**